packages/framework/synthesize/src/test/dependencyContainer.spec.ts

```diff
@@ -51,7 +50,7 @@ describe("Routerlicious", () => {
                 const loadable = await s.IFluidLoadable;
                 assert(loadable, "Optional IFluidLoadable was registered");
                 assert(loadable === mock, "IFluidLoadable is expected");
-                assert(loadable?.url === mock.url, "IFluidLoadable is valid");
+                assert(loadable?.handle.absolutePath === mock.handle.absolutePath, "IFluidLoadable is valid");
```

Comment:
Is there a longer term plan for these tests specifically?  If `handle.absolutePath` is just a temporary solution from the BREAKING.md changes, it may be helpful if this were to be fixed the correct "long-term" way.  One of the common hiccups during version bumps from Bohemia I've seen is that they take the temporary fix and it never transitions into the long term fix, then when the temporary fix is removed the OCE has no idea how to make the actual fix (and frequently the note in the BREAKING.md does not provide instructions on that either)

Reply:
To be honest, I do not know what's the plan for these tests :)
The wording is not really great in breaking.md - changing it. I removed any mentioning about future plans, as this is still TBD.